src/Main.cpp

```diff
@@ -324,25 +204,28 @@ int main(int argc, char** argv) {
     bsp.initChip((void*)&cmdLineArgs);
 
     static BittyBuzzTask s_bittybuzzTask("bittybuzz", tskIDLE_PRIORITY + 1);
-    //    static UartMessageDispatcher s_uartDispatchTask("uart_dispatch", tskIDLE_PRIORITY + 1);
-    static TCPMessageDispatcher s_tcpDispatchTask("tcp_dispatch", tskIDLE_PRIORITY + 1);
-    static SpiMessageDispatcher s_spiDispatchTask("spi_dispatch", tskIDLE_PRIORITY + 1);
-    static UartMessageSender s_uartMessageSender("uart_send", tskIDLE_PRIORITY + 1);
-    static TCPMessageSender s_tcpMessageSender("uart_send", tskIDLE_PRIORITY + 1);
-    static SPIMessageSender s_spiMessageSender("spi_send", tskIDLE_PRIORITY + 1);
-    static USBMessageSender s_usbMessageSender("usb_send", tskIDLE_PRIORITY + 5);
-    static USBMessageDispatcher s_usbMessageDispatcher("usb_receive", tskIDLE_PRIORITY + 5);
     static InterlocTask s_interlocTask("interloc", tskIDLE_PRIORITY + 5);
 
+    static MessageDispatcherTask s_hostDispatchTask("tcp_dispatch", tskIDLE_PRIORITY + 1, NULL,
```

Comment:
Shouldn't we keep tsk priorities defined somewhere? instead of + X after idle

Reply:
Added define at the top of the main